Neos.Flow/Classes/Mvc/ActionRequest.php

```diff
@@ -47,9 +47,9 @@ class ActionRequest implements RequestInterface
 
     /**
      * Package key of the controller which is supposed to handle this request.
-     * @var string
+     * @var string|null
```

Comment:
Note: unfortunately, packageKey and controllerName need to stay nullable for now, because we depend on PHP's default method argument behaviour inside `Controller::forward` and `UriBuilder` (a provided `null` value falling back to the default value in the method declaration). On the other hand, we have a lot of places where we just directly use the return values without checking for null and operate on it as a string.

At least, now it's explicit in the type hints and doc blocks.